app/mailboxes/comments_mailbox.rb

```diff
@@ -0,0 +1,21 @@
+class CommentsMailbox < ApplicationMailbox
+  before_processing :ensure_ticket
```

Comment:
I'd keep the naming similar to the one in the application_mailbox file. Something like `ensure_ticket_presence` or `ensure_ticket_is_present`